<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pizza Maker 2000</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="PizzaEnumCombo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>150</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="LabelUOrder">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>40</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Your order:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="YourPizza">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LabelUPizza">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>290</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Your pizza:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="InfoCooking">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>110</y>
      <width>235</width>
      <height>165</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>235</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QComboBox" name="PizzaStyleEnum">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>150</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="LabelUOrder_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Style preference</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
